ProcessMaker/Http/Controllers/Api/V1_1/CaseController.php

```diff
@@ -83,6 +86,73 @@ public function getCompleted(CaseListRequest $request): JSonResponse
         return $this->paginateResponse($query);
     }
 
+    /**
+     * Get "my cases" counters
+     *
+     * @param CaseListRequest $request
+     *
+     * @return JsonResponse
+     */
+    public function getMyCasesCounters(CaseListRequest $request): JsonResponse
+    {
+        // Load user object
+        if ($request->filled('userId')) {
+            $userId = $request->get('userId');
+            $user = User::find($userId);
+        } else {
+            $user = Auth::user();
+        }
+
+        // Initializing variables
+        $totalAllCases = null;
+        $totalMyCases = null;
+        $totalInProgress = null;
+        $totalCompleted = null;
+        $totalMyRequest = null;
+
+        // Check permission
+        if ($user->hasPermission('view-all_cases')) {
+            // The total number of cases recorded in the platform. User Id send is overridden.
+            $request->merge(['userId' => null]);
+            $queryAllCases = $this->caseRepository->getAllCases($request);
+            $totalAllCases = $queryAllCases->count();
+        }
+
+        // Restore user id
+        $request->merge(['userId' => $user->id]);
+
+        // The total number of cases recorded by the user making the request.
+        $queryMyCases = $this->caseRepository->getAllCases($request);
+        $totalMyCases = $queryMyCases->count();
+
+        // The number of In Progress cases started by the user making the request.
+        $queryInProgressCases = $this->caseRepository->getInProgressCases($request);
+        $totalInProgress = $queryInProgressCases->count();
+
+        // The number of Completed cases started by the user making the request.
+        $queryCompletedCases = $this->caseRepository->getCompletedCases($request);
+        $totalCompleted = $queryCompletedCases->count();
+
+        // Check permission
+        if ($user->hasPermission('view-my_requests')) {
+            // Only in progress requests
+            $requestAux = new Request();
+            $requestAux->replace(['type' => 'in_progress']);
+
+            // The number of requests for user making the request.
+            $totalMyRequest = (new ProcessRequestController)->index($requestAux, true, $user);
```

Comment:
As note, the query generated for this counter during the tests were:
```
select count(*) from `process_requests` where (`user_id` = 4 or 0 = 1 or 0 = 1) and `status` = 'ACTIVE' and `process_id` not in (1)
```
Which is the expected.